src/utils/peopleHelper.ts

```diff
@@ -0,0 +1,65 @@
+import { Person } from '../types';
+import { getCentury } from './functionHelpers';
+
+const sortPeopleWithSearch = (people: Person[], search: URLSearchParams) => {
+  const sort = search.get('sort') as keyof Person | null;
+  const order = search.get('order');
+
+  switch (sort) {
+    case 'name':
+      return people.sort((a, b) =>
+        order ? b.name.localeCompare(a.name) : a.name.localeCompare(b.name),
+      );
+    case 'sex':
+      return people.sort((a, b) =>
+        order ? b.sex.localeCompare(a.sex) : a.sex.localeCompare(b.sex),
+      );
+    case 'born':
+      return people.sort((a, b) => (order ? b.born - a.born : a.born - b.born));
+    case 'died':
+      return people.sort((a, b) => (order ? b.died - a.died : a.died - b.died));
```

Comment:
```suggestion
    case 'born':
    case 'died':
      return people.sort((a, b) => (order ? b[sort]- a[sort] : a[sort] - b[sort]));
```
you can combine some cases